Gems/EMotionFX/Code/Source/Editor/Plugins/Ragdoll/RagdollColliderTranslationManipulators.cpp

```diff
@@ -0,0 +1,193 @@
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Component/ComponentApplicationBus.h>
+#include <AzFramework/Physics/Character.h>
+#include <EMotionFX/CommandSystem/Source/ColliderCommands.h>
+#include <EMotionFX/Source/Actor.h>
+#include <EMotionFX/Source/Node.h>
+#include <EMotionStudio/EMStudioSDK/Source/EMStudioManager.h>
+#include <Editor/ColliderContainerWidget.h>
+#include <Editor/ObjectEditor.h>
+#include <Editor/Plugins/Ragdoll/RagdollColliderTranslationManipulators.h>
+
+namespace EMotionFX
+{
+    RagdollColliderTranslationManipulators::RagdollColliderTranslationManipulators()
+        : m_translationManipulators(
+              AzToolsFramework::TranslationManipulators::Dimensions::Three, AZ::Transform::CreateIdentity(), AZ::Vector3::CreateOne())
+    {
+        m_adjustColliderCallback = new DataChangedCallback(this, false);
+        EMStudio::GetCommandManager()->RegisterCommandCallback("AdjustCollider", m_adjustColliderCallback);
+    }
+
+    RagdollColliderTranslationManipulators::~RagdollColliderTranslationManipulators()
+    {
+        EMStudio::GetCommandManager()->RemoveCommandCallback(m_adjustColliderCallback, false);
+        delete m_adjustColliderCallback;
+    }
+
+    void RagdollColliderTranslationManipulators::Setup(RagdollManipulatorData& ragdollManipulatorData)
+    {
+        m_ragdollManipulatorData = ragdollManipulatorData;
+
+        if (!m_ragdollManipulatorData.m_valid || !m_ragdollManipulatorData.m_colliderNodeConfiguration ||
+            m_ragdollManipulatorData.m_colliderNodeConfiguration->m_shapes.empty())
+        {
+            return;
+        }
+
+        m_translationManipulators.SetSpace(m_ragdollManipulatorData.m_nodeWorldTransform);
+        m_translationManipulators.SetLocalPosition(m_ragdollManipulatorData.m_colliderNodeConfiguration->m_shapes[0].first->m_position);
+        m_translationManipulators.Register(EMStudio::g_animManipulatorManagerId);
+        AzToolsFramework::ConfigureTranslationManipulatorAppearance3d(&m_translationManipulators);
+
+        // mouse down callbacks
+        m_translationManipulators.InstallLinearManipulatorMouseDownCallback(
+            [this](const AzToolsFramework::LinearManipulator::Action& action)
+            {
+                BeginEditing(action.m_start.m_localPosition, action.m_current.m_localPositionOffset);
+            });
+
+        m_translationManipulators.InstallPlanarManipulatorMouseDownCallback(
+            [this](const AzToolsFramework::PlanarManipulator::Action& action)
+            {
+                BeginEditing(action.m_start.m_localPosition, action.m_current.m_localOffset);
+            });
+
+        m_translationManipulators.InstallSurfaceManipulatorMouseDownCallback(
+            [this](const AzToolsFramework::SurfaceManipulator::Action& action)
+            {
+                BeginEditing(action.m_start.m_localPosition, action.m_current.m_localOffset);
+            });
```

Comment:
question: Do we definitely want the `SurfaceManipulator` for these tools? Will it work as the user expects? It might be possible to IIRC to customize the view so we don't add the `SurfaceManipulator` (we do this for the `PolygonPrismShape` for example as the vertex can only move on the plane)

Reply:
It seems like the surface manipulator always gets created if you use 3 for the dimensions.